docs/operations/operations_on_Arm64.md

```diff
@@ -0,0 +1,31 @@
+# Operations
+
+This page summarizes all operations that are not supported on Arm64.
+
+## Hotplug Network Interfaces
+
+Hotplug Network Interfaces does not support on Arm64, because the image ghcr.io/k8snetworkplumbingwg/multus-cni:snapshot-thick does not support for Arm64 platform. For more information please refer to https://github.com/k8snetworkplumbingwg/multus-cni/pull/1027.
```

Comment:
```suggestion
Hotplug Network Interfaces are not supported on Arm64, because the image ghcr.io/k8snetworkplumbingwg/multus-cni:snapshot-thick does not support the Arm64 platform. For more information please refer to https://github.com/k8snetworkplumbingwg/multus-cni/pull/1027.
```

Checking my understanding: the image does not support Arm64?

Reply:
Yes, you are right.